### DISCUSSION TITLE
is it Possible to link optl collector (opentelemetry-collector-contrib. ) output to cloud trace for visualization  - GCP?

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [open-telemetry/opentelemetry-demo#887] [General]
I have now installed , operator, collector, auto-instrument for java and annotated the pod.

I can see on Collector LOG Trace and SPAN getting generated.

My Q sorry a very nave one. how will I visualise this? we are in GKE and Google does provide GOOGLE CLOUD TRACE>

Is there any way I can expose the date from collector to CLOUD TRACE at all?

**Comment by julianocosta89** [comment]
Hello @sreejesh-radhakrishnan-db, this question is not related to the opentelemetry-demo repository, so I'll close it.
But in order to give you some guidance, you can take a look at the `googlecloudexporter` https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/googlecloudexporter.
It can be used as an exporter within your OTelCollector config and send traces to Google Cloud Traces.
